src/render_graph.rs

```diff
@@ -49,23 +99,30 @@ pub fn render_graph_dot(graph: &RenderGraph) -> String {
                     input_node,
                     input_index,
                     output_node,
-                    output_index: _,
+                    output_index,
                 } => {
                     let input = graph.get_node_state(*input_node).unwrap();
                     let input_slot = &input.input_slots.iter().nth(*input_index).unwrap().info;
-                    let label = format!("\"{}\"", input_slot.name);
 
                     dot.add_edge(
                         &node_id(output_node),
+                        Some(&format!("{}:e", output_index)),
                         &node_id(input_node),
-                        &[("label", &label), edge_color],
+                        Some(&format!("{}:w", input_index)),
+                        &[edge_color],
                     );
                 }
                 Edge::NodeEdge {
                     input_node,
                     output_node,
                 } => {
-                    dot.add_edge(&node_id(output_node), &node_id(input_node), &[]);
+                    dot.add_edge(
+                        &node_id(output_node),
+                        Some("title:e"),
+                        &node_id(input_node),
+                        Some("title:w"),
+                        &[("style", "dashed")],
```

Comment:
```suggestion
                        &[],
```
Same as above, just a personal preference

Reply:
Done.